File: 2016/fall/8/week1/floats.py
import cs50

# prompt user for x
print("x is ", end="")
x = cs50.get_float()

# prompt user for y
print("y is ", end="")
y = cs50.get_float()

# perform division for user
print("{} divided by {} is {}".format(x, y, x / y))

File: 2017/fall/1/floats.c
// Floating-point arithmetic

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for x
    float x = get_float("x: ");

    // Prompt user for y
    float y = get_float("y: ");

    // Perform division
    printf("%f divided by %f is %f\n", x, y, x / y);
}
